src/main/java/com/flightstats/hub/webhook/WebhookService.java

```diff
@@ -42,6 +42,9 @@ public WebhookService(@Named("Webhook") Dao<Webhook> webhookDao, WebhookValidato
         webhook = webhook.withDefaults();
         webhookValidator.validate(webhook);
         Optional<Webhook> preExisting = get(webhook.getName());
+        if (webhook.isTagPrototype()) {
+            return upsertTagWebhook(webhook, preExisting);
+        }
```

Comment:
Should this be if else?

Reply:
else is not necessary because it is a method exit point